src/jdk.compiler/share/classes/com/sun/tools/javac/util/ByteBuffer.java

```diff
@@ -56,8 +56,13 @@ public ByteBuffer() {
      *  of given size.
      */
     public ByteBuffer(int initialSize) {
-        elems = new byte[initialSize];
-        length = 0;
+        this(new byte[initialSize]);
+    }
+
+    /** Create a new byte buffer using the given array for storage.
```

Comment:
Generally, the style for doc comments is either a single line, or for a newline after the initial `/**`

Reply:
Yeah... except not in this file. I was erring on the side of being consistent with the rest of the file (lesser of two evils?)

Reply:
OK, as long as you made a deliberate choice, that's cool.